api/src/db.js

```diff
@@ -97,6 +102,11 @@ if (UNIT_TEST_ENV) {
     }
   };
 
+  module.exports.wipeCacheDb = async () => {
+    await module.exports.cache.destroy();
```

Comment:
Tested this locally, and it deletes the database and also frees us used disk space.